src/components/Gradebook/gradebook.scss

```diff
@@ -41,24 +45,37 @@
 .gbook { 
     overflow-x: scroll;
 
+    .grade-button {
+        text-decoration: underline;
+    }
+
+    .student_key {
+        font-size: 14px;
+    }
+
     .table {
         padding-left: 244px;
+
+        th {
+            vertical-align: top;
```

Comment:
UX/UI request

"Table head row: suggest vertical alignment top"
"Table head row: suggest slightly smaller for font size, 14px. I realize this is smaller than our standard 16px, but I think it's necessary given how much text we are squeezing into each cell"